File: 백준/Bronze/5532. 방학 숙제/방학 숙제.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int sum = Integer.parseInt(br.readLine());
        int korean = Integer.parseInt(br.readLine());
        int math = Integer.parseInt(br.readLine());
        int maxKor = Integer.parseInt(br.readLine());
        int maxMath = Integer.parseInt(br.readLine());
        int korDay = (korean % maxKor == 0) ? korean / maxKor : korean / maxKor + 1;
        int mathDay = (math % maxMath == 0) ? math / maxMath : math / maxMath + 1;
        if(korDay > mathDay) System.out.println(sum - korDay);
        else System.out.println(sum - mathDay);
    }
}

File: 백준/Bronze/5532. 방학 숙제/방학 숙제.cc
#include <stdio.h>

int main(void) {
	int sum, korean, math, maxKor, maxMath;
	scanf("%d %d %d %d %d", &sum, &korean, &math, &maxKor, &maxMath);
	int korDay = korean / maxKor;
	if (korean % maxKor != 0) korDay++;
	int mathDay = math / maxMath;
	if (math % maxMath != 0) mathDay++;
	if (korDay > mathDay) printf("%d", sum - korDay);
	else printf("%d", sum - mathDay);
	return 0;
}